### PR TITLE
Run3-gex142B Use tokens rather than labels in accessing collections in Alignment/CommonAlignmentProducer

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Use tokens rather than labels in accessing collections in Alignment/CommonAlignmentProducer

#### PR validation:

Use the runTheMatrix test workflows

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Alignment/CommonAlignmentProducer/plugins/GlobalTrackerMuonAlignment.cc [hunk 1]
```diff
@@ -600,27 +629,21 @@ void GlobalTrackerMuonAlignment::analyzeTrackTrack(const edm::Event& iEvent, con
     std::cout << std::endl;
```
Comment (ChrisMisan) [inline Alignment/CommonAlignmentProducer/plugins/GlobalTrackerMuonAlignment.cc:629]: there are std::cout in the modified files, it is advised to change them to the edm::LogVerbatim

**Review by ChrisMisan — COMMENTED** [review]
It is advised to add fillDescriptions to the modified modules

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by ChrisMisan** [comment]
+alca

- diffs only in msgLogger
- recommendations were taken into account

**Comment by perrotta** [comment]
+1

**Pull request merged**